besu/src/main/java/org/hyperledger/besu/cli/options/stable/NodePrivateKeyFileOption.java

```diff
@@ -22,6 +22,8 @@
 
 /** The Node private key file Cli option. */
 public class NodePrivateKeyFileOption {
+  /** Default constructor. */
+  NodePrivateKeyFileOption() {}
```

Comment:
Why are some of them package-private?

Reply:
The private constructors should be "injected" in factories which only has static methods.

Reply:
In other words, if I am confident that the constructor should be declared private, I declare it as such. Otherwise if its an abstract class, protected constructor makes sense. package level access constructors are suitable where those constructors are utilized in picocli/unit tests.